### DISCUSSION TITLE
Сортировка треков в плейлисте по дате добавления в этот плейлист

### DISCUSSION THREAD

**Discussion by supilot** [Chimildic/YaMuTools#40] [Идея] [answered]
Удобная функция в Спотифае, которая позволяет вытащить новинки наверх в обновляемом плейлисте.

**Comment by regulator1992** [comment]
такой функции как я понял нет?

**Reply by VladWinner** [reply]
Она есть (Коллекция→Настройки→Прочее→«Добавлять треки в начало плейлиста»), но проблема в том, что она действует только при добавлении новых треков и не сказывается на ранее добавленных. 
Если вы, например, перенесли музыку из Spotify, то в импортированных плейлистах последнее добавленные треки будут в самом конце списка, и это никак не исправить, только **сидеть вручную** перемещать треки.
Если включили опцию и начали добавлять треки, они будут идти в начале, но старые треки так и останутся в другом порядке.
![image](https://user-images.githubusercontent.com/13555921/163731568-e0865a30-ae89-453a-ae14-39ae75ef959d.png)
По сути эту идею иначе можно сформулировать как «добавить опцию сортировки в обратном порядке, нежели сейчас».

**Comment by Chimildic** [comment, marked as answer]
API сайта не дает дату добавления трека. 
Добавил сортировку в обратном порядке в версии 0.8.0.
